app/controllers/submissions_controller.rb

```diff
@@ -28,8 +28,14 @@ def create
     @author_id = params[:author_id]
     if @submission.save
       if @author_id
-        authorship = Authorship.new(person_id: @author_id,submission: @submission)
-        authorship.save
```

Comment:
Ale dlaczego to zostało usunięte i zastąpione tymi linijkami poniżej??? To był poprawny kod, a teraz to nie ma sensu.